tests/test.js

```diff
@@ -0,0 +1,7 @@
+import { expect } from 'chai';
+
+describe('Addtion', () => {
+  it('shoud add two numbers', () => {
```

Comment:
Please pay more attention to details
```suggestion
  it('should add two numbers', () => {
```